.github/workflows/pull-request.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   changedfiles:
     name: Publish to docs
-    runs-on: [ubuntu-latest]
+    runs-on: [self-hosted, services, X64]
```

Comment:
?????

Reply:
These are the runners all the other FE repos use. I wanted to see if fixing this fixed the docs PR error.

Reply:
I wouldn't use private runners on an open-source repository. Let's keep `ubuntu-latest`.

Reply:
We attempt to create a PR in the docs repo when changes are made in this repo. Here is the error: https://github.com/moovfinancial/moov-node/actions/runs/3483437814/jobs/5826924611

Reply:
GITHUB_TOKEN only has permissions for the repository it's running in and forks have even less access. 

https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
I'm not sure if giving an oss repository access to write to docs.moov.io is the best idea. Can we run a scheduled job in docs to pull the latest moov-node code?

Reply:
Yes, we'll explore that and possibly other ways to do this. Thanks, @adamdecaf!